crates/kubelet/src/node/mod.rs

```diff
@@ -0,0 +1,5 @@
+#![allow(missing_docs)]
+
+mod node;
+
+pub use node::{create, drain, evict_pods, update, Builder, Node};
```

Comment:
For my own edification, this is to make the publicly exported node functions more clear (e.g. make them stand out from the huge file), correct?

Reply:
Bingo

Splitting them out into separate files makes it simpler to split out work into separate modules in case we need to implement further node API calls in the future (node resource stats, for example?)